From b1d258c832a9dba86e6dbd7d230c4bb8e1d60ecf Mon Sep 17 00:00:00 2001
From: Josh Morman <mormjb@gmail.com>
Date: Mon, 20 Apr 2020 08:21:16 -0400
Subject: [PATCH 04/32] blocks: update examples for chooser default

---
 gr-blocks/examples/matrix_multiplexer.grc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gr-blocks/examples/matrix_multiplexer.grc b/gr-blocks/examples/matrix_multiplexer.grc
index c266c7a412..9cff0d2a7e 100644
--- a/gr-blocks/examples/matrix_multiplexer.grc
+++ b/gr-blocks/examples/matrix_multiplexer.grc
@@ -43,6 +43,7 @@ blocks:
     label4: ''
     labels: '[]'
     num_opts: '3'
+    option0: ((1, 0), (0, 1))
     option1: ((0, 1), (1, 0))
     option2: ((.5, .5), (.5, .5))
     option3: '3'
-- 
2.20.1

