From 1591d414c842d561dd209cb4d615219749bfc203 Mon Sep 17 00:00:00 2001
From: Sebastian Koslowski <sebastian.koslowski@gmail.com>
Date: Sat, 16 May 2020 22:00:03 +0200
Subject: [PATCH 17/32] grc: fixup: dynamically update namespaces from blocks
 (#3317)

---
 grc/core/blocks/block.py | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/grc/core/blocks/block.py b/grc/core/blocks/block.py
index a3c724c3fb..e738e36cf7 100644
--- a/grc/core/blocks/block.py
+++ b/grc/core/blocks/block.py
@@ -87,6 +87,7 @@ class Block(Element):
         self.active_sinks = []  # on rewrite
 
         self.states = {'state': True, 'bus_source': False, 'bus_sink': False, 'bus_structure': None}
+        self.block_namespace = {}
 
         if 'cpp' in self.flags:
             self.orig_cpp_templates = self.cpp_templates # The original template, in case we have to edit it when transpiling to C++         
@@ -139,17 +140,18 @@ class Block(Element):
         self.active_sinks = [p for p in self.sinks if not p.hidden]
 
         # namespaces may have changed, update them
-        self.block_namespace = {}
+        self.block_namespace.clear()
+        imports = ""
         try:
-            exec(self.templates.render('imports'), self.block_namespace)
+            imports = self.templates.render('imports')
+            exec(imports, self.block_namespace)
         except ImportError:
             # We do not have a good way right now to determine if an import is for a
             # hier block, these imports will fail as they are not in the search path
             # this is ok behavior, unfortunately we could be hiding other import bugs
             pass
         except Exception:
-            log.exception('Failed to evaluate import expression "{0}"'.format(expr), exc_info=True)
-            pass
+            self.add_error_message(f'Failed to evaluate import expression {imports!r}')
 
     def update_bus_logic(self):
         ###############################
-- 
2.20.1

